nestkernel/simulation_manager.h

```diff
@@ -151,13 +145,15 @@ class SimulationManager : public ManagerInterface
   delay to_step_;         //!< update clock_+from_step<=T<clock_+to_step_
   timeval t_slice_begin_; //!< Wall-clock time at the begin of a time slice
   timeval t_slice_end_;   //!< Wall-clock time at the end of time slice
-  long t_real_;    //!< Accumunated wall-clock time spent simulating (in us)
-  bool terminate_; //!< Terminate on signal or error
+  long t_real_;     //!< Accumunated wall-clock time spent simulating (in us)
+  bool simulating_; //!< true if simulation in progress
```

Comment:
Same state machine comment here: what are the possible states of the object? Maybe a single enum is needed here with the state nodes, and a description of the transitions between nodes?